addons/mail/static/src/messaging/component/partner_im_status_icon/partner_im_status_icon.scss

```diff
@@ -15,6 +15,12 @@
     transform: scale(1.5);
 }
 
+.o-background {
+    transform: scale(1);
+    margin-inline-end: map-get($spacers, 1);
+    margin-top: 2px;
```

Comment:
spacers here as well

Reply:
In what way is it ugly? I would expect spacers 1 to be equivalent to 2px? It would be strange it that is not the case here.